packages/components/pages/f-checkout/src/components/Notes.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+    <div v-if="isSplitNotesEnabled">
+        <accordion
+            :id="noteTypeCourierOrOrder"
+            :title="$t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.title`)">
+            <span :class="$style['c-checkout-accordion-help']">{{ $t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.text`) }}</span>
+            <form-field
+                :placeholder="$t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.placeholder`)"
+                :value="noteValue"
+                v-bind="inputStyles"
+                :name="`${noteTypeCourierOrOrder}-note`"
+                @input="updateUserNotes({ note: $event, type: noteTypeCourierOrOrder })" />
+        </accordion>
+        <accordion
+            v-if="shouldShowKitchenNotes"
+            id="kitchen"
+            :title="$t(`userNote.kitchen.${serviceType}.title`)">
+            <span :class="$style['c-checkout-accordion-help']">{{ $t(`userNote.kitchen.${serviceType}.text`) }}</span>
+            <form-field
+                :placeholder="$t(`userNote.kitchen.${serviceType}.placeholder`)"
+                :value="kitchenNoteValue"
+                v-bind="inputStyles"
+                name="kitchen-note"
+                @input="updateUserNotes({ note: $event, type: 'kitchen' })" />
+        </accordion>
+    </div>
+    <form-field
+        v-else-if="!isSplitNotesEnabled"
```

Comment:
Would simply `v-else` work here or does the `v-if` on L15 interfere?

Reply:
That works, but I guess my fear was that what you said would be the case. Now changed.